cluster/src/main/java/io/scalecube/cluster/membership/MembershipProtocol.java

```diff
@@ -281,6 +281,8 @@ public void stop() {
         .timeout(config.getSyncTimeout(), TimeUnit.MILLISECONDS, scheduler)
         .subscribe(
             message -> {
+              SyncData syncData = message.data();
```

Comment:
Also you should not have this diff as well, since this was added to master already